src/components/dashboard/user-settings-menu.tsx

```
@@ -0,0 +1,203 @@
+  Users,
+  MessageSquare,
+  LogOut,
+  Monitor,
+  Sun,
+  Moon,
+  ChevronRight,
+  Check,
+} from 'lucide-react'
+import Link from 'next/link'
+import { useTheme } from 'next-themes'
+import { useLocale } from 'next-intl'
```

ESLint

[error] `next-intl` import should occur before import of `next-themes`

(import/order)